V5_MIGRATION.md

```diff
@@ -16,23 +16,58 @@ v5 introduces limited support for Swift Package Manager. See the [README](/READM
 
 In v4, 3D Secure classes were housed in the `BraintreePaymentFlow` module. In v5, `BraintreeThreeDSecure` is a standalone module offering the same 3DS functionality. The `BraintreePaymentFlow` module still houses Local Payment functionality.
 
-### CocoaPods
+#### Integration
```

Comment:
```suggestion
### Integration
```

Reply:
Another nitpick - changing these from #### to ###.

Reply:
I intentionally used four #'s because I thought it made it more clear that these are subsections of 3D Secure. It just looks cleaner to me visually. I think I'll leave them if you're OK with it.